src/runtime-tools/linux/setup.sh

```diff
@@ -26,6 +27,7 @@ mkdir -p /onefuzz/setup
 mkdir -p /onefuzz/tools
 mkdir -p /onefuzz/etc
 mkdir -p /onefuzz/instance-specific-setup
+mkdir -p /onefuzz/tools/dotnet
```

Comment:
```suggestion
mkdir -p "$DOTNET_ROOT"
```

reuse existing variable